### PR TITLE
Smaller line length

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: test/check-machines-create [hunk 1]
```diff
@@ -174,22 +174,24 @@ class TestMachinesCreate(machineslib.VirtualMachinesCase):
...
 
         # user login
+        errors = {"create-vm-dialog-user-login": "User login must not be empty when user password is set"}
         runner.checkDialogFormValidationTest(TestMachinesCreate.VmDialog(self, sourceType='cloud',
                                                                          storage_size=10, storage_size_unit='MiB',
                                                                          location=config.VALID_DISK_IMAGE_PATH,
                                                                          user_password="catsaremybestfr13nds",
                                                                          create_and_run=True),
-                                             {"create-vm-dialog-user-login": "User login must not be empty when user password is set"})
+                                                                        errors)
```
Comment (martinpitt) [inline test/check-machines-create:183]: This indentation change is confusing -- it now looks like `errors` is an argument to `VmDialaog()`, but it's not -- it's the second argument to `checkDialogFormValidationTest`. Same for the instances below.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Nice, thank you!

File: test/check-machines-create (hunk 1 above)
Comment (jelly) [reply test/check-machines-create]: Good catch.

**Review by martinpitt — APPROVED** [review]
*squeeze* *squeeze* Thanks!

**Pull request merged**